dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/AbstractOracleConnectorTest.java

```diff
@@ -0,0 +1,37 @@
+package com.google.edwmigration.dumper.application.dumper.connector.oracle;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.google.edwmigration.dumper.application.dumper.ConnectorArguments;
+import com.google.edwmigration.dumper.application.dumper.MetadataDumperUsageException;
+import java.lang.Runnable;
+import java.util.Properties;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class AbstractOracleConnectorTest {
+  // provide mocks only with a username and a password
+  // ignore the useFetchSize... property, because it has a default
+  private static final String EXPECTED_PASSWORD = "p4ssw0rd";
+  private static final String EXPECTED_USER = "user123";
+
+  private final ConnectorArguments arguments = mock(ConnectorArguments.class);
+  private final AbstractOracleConnector connector = new OracleMetadataConnector();
+
+  @Test
+  public void testValid() {
+    when(arguments.getPassword()).thenReturn(EXPECTED_PASSWORD);
```

Comment:
The variables `EXPECTED_PASSWORD` and `EXPECTED_USER` are used in two different places - here as **actual** values, later in the `assertEquals` call as the **expected** values. It might be worth replacing the `EXPECTED_` prefix with something unrelated to actual/expected wording, e.g. `SAMPLE_`.

Reply:
Done, changed to `EXAMPLE_...`